src/core/Akka/Actor/Internals/ActorSystemImpl.cs

```diff
@@ -127,7 +127,8 @@ public override ActorSelection ActorSelection(string actorPath)
 
         private void ConfigureScheduler()
         {
-            _scheduler = new DedicatedThreadScheduler(this);;// new TaskBasedScheduler();
+            var schedulerType = Type.GetType(_settings.SchedulerClass, true);
```

Comment:
mmm @rogeralsing the logger is not initialized until the system is Started, and that is too late to be able to log the scheduler failure.